### ISSUE TITLE
Consider re-adding snap support

### ISSUE THREAD

**Issue by atomicptr** [atomicptr/crab#9]
A few months ago the snap part of the pipeline broke and I haven't been able to get it working again since then so I decided to remove snap support for now (might still push updates manually we'll see...). 

Since I don't use Ubuntu nor Snap my interest in this is also kinda low anyway. If someone wants to help fix this I'm open to it.

**Comment by ramazansancar** [comment]
Has this issue been resolved? Is it still occurring? @atomicptr

**Comment by atomicptr** [comment]
Nope, still not available as a Snap, I don't have (nor want to use) Snap so if someone really wants the Snap support back they'd have to configure goreleaser themselves

**Comment by ramazansancar** [comment]
Let me see if anyone can help me with this :)
